eng/Directory.Packages.props

```diff
@@ -58,7 +58,6 @@
     <PackageVersion Include="Microsoft.Internal.VisualStudio.Shell.Framework" Version="17.9.35796-preview.1" />
     <PackageVersion Include="Microsoft.ServiceHub.Client" Version="4.2.1017" />
     <PackageVersion Include="Microsoft.VisualStudio.Extensibility" Version="$(MicrosoftVisualStudioExtensibilityVersion)" />
-    <PackageVersion Include="Microsoft.VisualStudio.Extensibility.Sdk" Version="$(MicrosoftVisualStudioExtensibilityVersion)" />
```

Comment:
Unused.

The SDK package combines Microsoft.VisualStudio.Extensibility + Microsoft.VisualStudio.Extensibility.JsonGenerators.Sdk. We need to reference them separately.

Reply:
> We need to reference them separately, so we can set PrivateAssets on the latter but not the former.

Is that a workaround for something?

Reply:
Not really. The SDK package is fine, it just won't work right on a project that you also need to create a nupkg from.